esp/esdllib/esdl_xpath_extensions_libxml.cpp

```diff
@@ -198,9 +367,19 @@ void registerEsdlXPathExtensions(IXpathContext *xpathContext, IEspContext *conte
         xpathContext->registerFunction(nullptr, "validateFeaturesAccess", (void  *)validateFeaturesAccessFunction);
         xpathContext->registerFunction(nullptr, "secureAccessFlags", (void  *)secureAccessFlagsFunction);
         xpathContext->registerFunction(nullptr, "getFeatureSecAccessFlags", (void  *)getFeatureSecAccessFlagsFunction);
+        xpathContext->registerFunction(nullptr, "getStoredStringValue", (void  *)getStoredStringValueFunction);
+        xpathContext->registerFunction(nullptr, "storedValueExists", (void  *)storedValueExistsFunction);
+        xpathContext->registerFunction(nullptr, "getLogProfile", (void  *)getLogProfileFunction);
+        xpathContext->registerFunction(nullptr, "getLogOption", (void  *)getLogOptionFunction);
+        xpathContext->registerFunction(nullptr, "logOptionExists", (void  *)logOptionExistsFunction);
     }
 
     xpathContext->registerFunction("urn:hpcc:esdl:script", "validateFeaturesAccess", (void  *)validateFeaturesAccessFunction);
     xpathContext->registerFunction("urn:hpcc:esdl:script", "secureAccessFlags", (void  *)secureAccessFlagsFunction);
     xpathContext->registerFunction("urn:hpcc:esdl:script", "getFeatureSecAccessFlags", (void  *)getFeatureSecAccessFlagsFunction);
+    xpathContext->registerFunction("urn:hpcc:esdl:script", "getStoredStringValue", (void  *)getStoredStringValueFunction);
+    xpathContext->registerFunction("urn:hpcc:esdl:script", "storedValueExists", (void  *)storedValueExistsFunction);
+    xpathContext->registerFunction("urn:hpcc:esdl:script", "getLogProfile", (void  *)getLogProfileFunction);
+    xpathContext->registerFunction("urn:hpcc:esdl:script", "getLogOption", (void  *)getLogOptionFunction);
+    xpathContext->registerFunction("urn:hpcc:esdl:script", "logOptionExists", (void  *)logOptionExistsFunction);
```

Comment:
This looks like a candidate for a helper function, passing the urn and registering all functions. As the function list grows it will be more helpful.

Reply:
agreed the list is getting long enough.